static/app/components/datePageFilter.tsx

```diff
@@ -58,6 +58,7 @@ function DatePageFilter({router, resetParamsOnChange, ...props}: Props) {
         hideBottomBorder={false}
         isOpen={isOpen}
         highlighted={desyncedFilters.has('datetime')}
+        data-test-id="global-header-timerange-selector"
```

Comment:
In the future let's probably rename these

Reply:
yup, will clean up in future PR